compose/src/main/java/com/maplibre/compose/symbols/Polyline.kt

```diff
@@ -24,16 +29,35 @@ import com.maplibre.compose.ramani.PolyLineNode
 @MapLibreComposable
 fun Polyline(
     points: List<LatLng>,
-    color: String,
+    color: String? = null,
```

Comment:
Is `null` a good default? `color` doesn't do anything if a `linePatternId` is specified...

Reply:
Let's document